spec/requests/issues_controller_spec.rb

```diff
@@ -141,6 +142,33 @@
         ),
       )
     end
+
+    describe "group access" do
+      before { SiteSetting.discourse_jira_allowed_groups = Group::AUTO_GROUPS[:moderators]
+               Group.refresh_automatic_groups!
+              }
+
+      describe "regular user with insufficient permissions" do
+        it "does not allow access" do
+          sign_in(user)
+
+          post "/jira/issues.json"
```

Comment:
Also you may know this already but you will need to add `params` for this call and the one below